NOTICE

```diff
@@ -71,6 +71,7 @@ The text of each license is also included in licenses/LICENSE-[project].txt
 * hat-trie(https://github.com/Tessil/hat-trie)
 * pegtl(https://github.com/taocpp/PEGTL, NOTE: changed to Boost Software License Version 1.0 in main branch)
 * cpptrace(https://github.com/jeremy-rifkin/cpptrace)
+* spdlog(https://github.com/gabime/spdlog)
```

Comment:
We should remove the `glog` from the NOTICE?

Reply:
Done.